plugins/github/api.ml

```diff
@@ -646,13 +662,30 @@ module Refs = Monitor(struct
     let hash = node / "headRefOid" |> to_string in
     let pr = node / "number" |> to_int in
     let title = node / "title" |> to_string in
+    let headRefName = node / "headRefName" |> to_string in
```

Comment:
I see that the names `headRefName` and `headNameWithOwner` come from the GitHub API, but would it be nicer to use something like `branch_name` and `fork`? It could be clearer to map to the names from the API in some cases, but I'm leaning towards the simpler names.

Reply:
tl;dr: I'm not sure the naming here matters too much... but I am mostly not sure :)

The suggestions here seem to fit with the naming of the accessor functions. And I can see value in keeping naming between the fields and accessor functions aligned. On the other hand, naming the field after the gql key follows the precedent of `bodyHTML`.

But then we don't follow the same practice here with `base` above. This leads me to think that we mabye don't have a consistent approach here between using the record as a more or less raw representation of the gql data vs. using it as a more refined/abstracted view.

Reply:
I agree, we probably don't have a consistent approach. But, it may be nicer to have `head` when we have a `base`.

Reply:
I changed the variables according to @punchagan, I also think they make sense (especially given that the names GitHub gave them are more obscure)